platform/ios/src/MGLMapView.mm

```diff
@@ -3589,7 +3589,7 @@ - (void)_setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count
         latLngs.push_back({coordinates[i].latitude, coordinates[i].longitude});
     }
     
-    CLLocationDirection cameraDirection = direction >= 0 ? direction : 0;
+    CLLocationDirection cameraDirection = direction >= 0 ? direction : self.direction;
```

Comment:
Could you please add a note in the method's documentation. Something along the lines:
> To keep the current direction pass a negative value...

Reply:
Sure. CLLocationDirection’s documentation states that a negative value means an invalid direction, but it is up to the MGLMapView documentation to state that an invalid direction is ignored when animating the camera.